AirCastingTests/ChooseSessionType/ChooseSessionTypeTests.swift

```diff
@@ -0,0 +1,43 @@
+// Created by Lunar on 04/08/2021.
+//
+
+import XCTest
+@testable import AirCasting
+import CoreLocation
+
+class ChooseSessionTypeTests: XCTestCase {
+
+    private var defaultSessionTypeViewModel: ChooseSessionTypeViewModel!
+    
+    override func setUp() {
+        defaultSessionTypeViewModel = ChooseSessionTypeViewModel(locationHandler: DefaultLocationHandler(locationTracker: LocationTracker(locationManager: CLLocationManager())), bluetoothHandler: DefaultBluetoothHandler(bluetoothManager: BluetoothManager(mobilePeripheralSessionManager: MobilePeripheralSessionManager(measurementStreamStorage: CoreDataMeasurementStreamStorage(persistenceController: PersistenceController.shared)))), userSettings: UserSettings(), sessionContext: CreateSessionContext(), urlProvider: UserDefaultsBaseURLProvider())
```

Comment:
Great to see you making first steps in unit testing!

Here you need to provide test doubles instead of real implementations.

Reply:
Do I need to create test doubles to every object? Do in my case for LocationTracker, BluetoothManager, CoreDataMeasurementStreaStorage etc. 😳?